maintenance/errors/error.go

```diff
@@ -219,5 +219,10 @@ func (e sentryEvent) build() *raven.Packet {
 	// from env
 	packet.Extra["microservice"] = os.Getenv("JAEGER_SERVICE_NAME")
 
+	// add logs
+	if sink, ok := log.SinkFromContext(ctx); ok {
+		packet.Extra["logs"] = sink.Logs()
```

Comment:
How is this handled by Sentry? Because it's a slice and not a string.

Reply:
It's handled the same way as the `oauth2.Scopes`. Please have a look in Sentry (for example pay) to see a concrete example.